rust/blockstore/src/arrow/block/delta/int32.rs

```diff
@@ -0,0 +1,125 @@
+use super::{calculate_key_size, calculate_prefix_size, BlockKeyArrowBuilder};
```

Comment:
nit: filename should be int32array since that is the type and not int32?

Reply:
this gets deleted in the stack above entirely so going to leave it